app/components/evergreen-field/evergreen-field-directive.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+angular.module('evergreenField.directive', [
+  'bulbsCmsApp.settings',
+  'lodash',
+  'saveButton.directive',
+])
+  .directive('evergreenField',
+    // TODO: akos -- this breaks when using new-style array, "routes" is undefined
+    function (routes) {
```

Comment:
Set it up like this:
```js
'use strict';

angular.module('evergreenField.directive', [
  'bulbsCmsApp.settings',
  'lodash',
  'saveButton.directive',
])
  .directive('evergreenField', [
    'routes',
    function (routes) {
      return {
        controller: [
          '_', '$scope', 'ContentFactory',
          function (_, $scope, ContentFactory) {
        }],
        restrict: 'E',
        scope: {
          content: '='
        },
        templateUrl: routes.COMPONENTS_URL + 'evergreen-field/evergreen-field.html'
      };
    }
  ]);
```

This way, angular will lookup that provider by string name, then inject it in the given position in the function arguments. Using this format prevents errors during minification and should save us some time on minifying sources.